cmd/contrib/main.go

```diff
@@ -180,15 +180,22 @@ func newPlugin() error {
 		}
 	}
 
-	// As a placeholder, assume the field name is the short version (max 7 chars) of the credential name, starting
-	// from the last word. For example:
+	// As a placeholder, assume the field name is the short version (max 7 chars) of the credential name, starting from the last word.
+	//
+	// For example:
 	// "Personal Access Token" => "Token"
 	// "Secret Key" => "Key"
 	// "API Key" => "API Key"
+	//
+	// When the last word of the credential name is greater than seven characters, the last word is used as the field name
 	var fieldNameSplit []string
 	lengthCutoff := 7
 	for i := range credNameSplit {
 		word := credNameSplit[len(credNameSplit)-1-i]
+		if i == 0 {
```

Comment:
small nit, but I would prefer something like:
```suggestion
        fieldNameSplit := []string{credNameSplit[len(credNameSplit) - 1]}
        if len(credNameSplit) > 1 {
	  for i := range credNameSplit[:len(credNameSplit) - 1] {
		  word := credNameSplit[len(credNameSplit)-1-i]
		  if len(strings.Join(append(fieldNameSplit, word), " ")) > lengthCutoff {
			  break
		  }
	  }
	}
```
or something along these lines. given that we want the last element _always_ in the field name, I find it clearer to already init the slice with this token in it.

Reply:
Thanks for the tip, @hculea! This has a tiny problem that "GitHub API Key" will result only in "Key" while we have space for "API Key" (cutoff is 7 characters). So, I have taken a slightly updated approach in https://github.com/1Password/shell-plugins/pull/263/commits/cb4a91f5d0190b3634057608a53e24ada513a63f. This should satisfy all of these cases:

- "Personal Access Token" => "Token"
- "Secret Key" => "Key"
- "API Key" => "API Key"
- "GitHub API Key" => "API Key"
- "Credentials" => "Credentials"